### DISCUSSION TITLE
Richeditbox RFT version

### DISCUSSION THREAD

**Discussion by miquik** [microsoft/WindowsAppSDK#1507] [Q&A] [answered]
Hi everybody,
Is still possible to use RichEditBox with WindowsAppSDK? If so, I would like to know what version of RTF specification is used

thanks

**Comment by andrewleader** [comment, marked as answer]
Hey @miquik, since this is a WinUI question, you can ask that on the WinUI repo: https://github.com/microsoft/microsoft-ui-xaml/issues. Thanks!
